### PR TITLE
Updates to plotting page: adds events table, fixes unclosed div tags,…

### PR DESCRIPTION
**Pull request by FBRTMaka**
… lines up event toggle button

@DanielJMaher , @birdage , @Bobfrat 

Updates to the plotting page to include the events table.

### CODE REVIEW

**Comment by FBRTMaka** [comment]
DO NOT MERGE YET

**Comment by FBRTMaka** [comment]
OK TO MERGE!!!

**Comment by FBRTMaka** [comment]
@DanielJMaher , don't forget about this!! goes with that last merge

**Comment by birdage** [comment]
@DanielJMaher looks like you didnt read it....

File: ooiui/static/js/partials/SVGPlotControl.html [hunk 1]
```diff
@@ -51,10 +51,12 @@
...
-        <label for="plotting-enable-events">Events</label>
-        <input class="bootstrap-switch btn btn-default" type="checkbox" id="plotting-enable-events" data-size="normal">        
-      </div>      
+        <div class="form-group" style="padding-top:0px;">
+            <label for="plotting-enable-events">Events</label>
+            <div>
+                <input class="bootstrap-switch btn btn-default" type="checkbox" id="plotting-enable-events" data-size="normal">        
+            </div>
+        </div>      
     </div>
```
Comment (FBRTMaka) [inline ooiui/static/js/partials/SVGPlotControl.html]: This lines up the event toggle button

File: ooiui/templates/science/plotting.html [hunk 2]
```diff
@@ -54,8 +54,8 @@
             </div> 
           </div> 
         </div> 
-
-      <!--  Plot section of the page   -->
+    </div>
+    <!--  Plot section of the page   -->
       <div id='bottom-row' class="row">
```
Comment (FBRTMaka) [inline ooiui/templates/science/plotting.html]: This lines up all the tables so they look tidy.

**Comment by DanielJMaher** [comment]
@birdage What is the issue?

**Comment by birdage** [comment]
no issue :+1: good to go!

**Comment by DanielJMaher** [comment]
@birdage confusing

**Pull request merged**